assets/js/app.js

```diff
@@ -42,6 +42,30 @@ var Helpers = function () {
                 textSplit.splice(pos, 1);
             }
             return textScrambled.join('');
+        },
+        jumbleText: function jumbleText(text) {
+            var textJumbled = [];
+            var nonWhitespaces = text.split(/\s+/).filter(function (v) { return v !== ''; });
+            var whitespaces = text.split(/[^\s]+/).filter(function (v) { return v !== ''; });
+
+            var longerLength = nonWhitespaces.length > whitespaces.length ? nonWhitespaces.length : whitespaces.length;
+            var pos;
+            for (var i = 0; i < longerLength; i++) {
+                // Add non-whitespace
+                if (nonWhitespaces.length > 0) {
+                    pos = Math.floor(Math.random() * nonWhitespaces.length);
+                    textJumbled.push(nonWhitespaces[pos]);
+                    nonWhitespaces.splice(pos, 1);
+                }
+
+                // Add whitespace
+                if (whitespaces.length > 0) {
+                    pos = Math.floor(Math.random() * whitespaces.length);
+                    textJumbled.push(whitespaces[pos]);
+                    whitespaces.splice(pos, 1);
+                }
+            }
+            return textJumbled.join('');
```

Comment:
could really be more efficient, but is simple to read and works for now. Does have a bit of lag.